src/parser/shared/modules/items/bfa/GildedLoaFigurine.js

```diff
@@ -31,16 +40,21 @@ class GildedLoaFigurine extends Analyzer {
     return this.selectedCombatant.getBuffUptime(SPELLS.WILL_OF_THE_LOA.id) / this.owner.fightDuration;
   }
 
-  item() {
-    return {
-      item: ITEMS.GILDED_LOA_FIGURINE,
-      result: (
-        <TooltipElement content={`Procced ${this.buffTriggerCount} times`}>
-          {formatPercentage(this.totalBuffUptime)}% uptime<br />
-          {formatNumber(this.totalBuffUptime * this.statBuff)} average {this.selectedCombatant.spec.primaryStat}
-        </TooltipElement>
-      ),
-    };
+  statistic() {
+    console.log(this.selectedCombatant.spec.primaryStat);
+    return (
+      <ItemStatistic
+        size="flexible"
+        tooltip={`Procced ${this.buffTriggerCount} times`}
+      >
+        <BoringItemValueText item={ITEMS.GILDED_LOA_FIGURINE}>
+          <UptimeIcon /> {formatPercentage(this.totalBuffUptime)}% uptime<br />
+          {this.selectedCombatant.spec.primaryStat === "Intellect" ? <><IntellectIcon /> {formatNumber(this.totalBuffUptime * this.statBuff)} <small>average intellect</small></> : ''}
```

Comment:
This can be shortened:

`<PrimaryStatIcon stat={this.selectedCombatant.spec.primaryStat} /> {formatNumber(this.averageStatModifier)} <small>average {this.selectedCombatant.spec.primaryStat} gained</small><`

Reply:
oh, didnt know about that. ill do that when i get home.